nbextensions/bulkImportCell/tabs/xsvGenerator.js

```diff
@@ -10,8 +10,102 @@ define(['StagingServiceClient', 'common/html', 'common/runtime', 'common/ui', 'u
         p = html.tag('p'),
         ul = html.tag('ul'),
         li = html.tag('li'),
+        select = html.tag('select'),
+        option = html.tag('option'),
+        input = html.tag('input'),
+        label = html.tag('label'),
+        fieldset = html.tag('fieldset'),
         cssBaseClass = 'kb-xsv-gen';
 
+    const formConfig = {
```

Comment:
I plan to use the standard kbase spec to generate and validate this form, although for this rough draft, I'm not using the params widgets -- I'm just generating form elements manually.